extensions/hdfs-storage/src/main/java/io/druid/storage/hdfs/HdfsDataSegmentPusher.java

```diff
@@ -57,6 +57,8 @@ public HdfsDataSegmentPusher(
     this.config = config;
     this.hadoopConfig = hadoopConfig;
     this.jsonMapper = jsonMapper;
+
+    log.info("Configured HDFS as deep storage");
```

Comment:
I prefer that it is not because often when I am debugging clusters I wish I had this log line